rust/gateway/src/messages.rs

```diff
@@ -10,10 +10,12 @@ use serde::{Deserialize, Serialize};
 use std::net::IpAddr;
 
 // TODO: Should this have a resource?
-#[derive(Debug, PartialEq, Eq, Deserialize, Serialize, Clone)]
+#[derive(Debug, PartialEq, Eq, Deserialize, Clone)]
 pub struct InitGateway {
     pub interface: Interface,
     pub config: Config,
+    #[serde(default)]
+    pub relays: Vec<Relay>,
```

Comment:
cc @AndrewDryga Same for the gateway, I am assuming that the field will be called `relays` and will be a sibling field of `interface` and `config`.